examples/streaming/wordcount-java/src/main/java/org/apache/gearpump/streaming/examples/wordcountjava/dsl/WordCount.java

```diff
@@ -46,15 +51,13 @@ public static void main(Config akkaConf, String[] args) throws InterruptedExcept
     JavaStream<String> sentence = app.source(new StringSource("This is a good start, bingo!! bingo!!"),
         1, UserConfig.empty(), "source");
 
-    JavaStream<String> words = sentence.flatMap(s -> Arrays.asList(s.split("\\s+")).iterator(),
-        "flatMap");
+    JavaStream<String> words = sentence.flatMap(new Split(), "flatMap");
 
-    JavaStream<Tuple2<String, Integer>> ones = words.map(s -> new Tuple2<>(s, 1), "map");
+    JavaStream<Tuple2<String, Integer>> ones = words.map(new Ones(), "map");
```

Comment:
With the addition of `setup` and `teardown` methods, Java 8 lambda functions no longer work.